docs/book/src/developer/providers/cluster-infrastructure.md

```diff
@@ -35,6 +35,11 @@ A cluster infrastructure provider must define an API type for "infrastructure cl
             `FailureDomainSpec` is defined as:
             - `controlPlane` (bool): indicates if failure domain is appropriate for running control plane instances.
             - `attributes` (`map[string]string`): arbitrary attributes for users to apply to a failure domain.
+7. Should have a `Status.Conditions` with the following:
+   1. A `Paused` condition to report if the cluster or cluster infrastructure is paused. It should check if 'spec.paused' is set on the cluster, and for the paused annotation on the infrastructure. This is currently optional, but will be required in the future.
+   ```go
+{{#include bootstrap.md:32:55}}
```

Comment:
Maybe lets move the text for paused in its own file and either just add a link here or embed it like you did. Currently it can break easily

Reply:
Sounds much more sensible :) Done!